spec/controllers/pages_controller_spec.rb

```diff
@@ -24,4 +24,21 @@ def index
       expect(response).to render_template(layout: false)
     end
   end
+
+  describe 'content expiry' do
```

Comment:
PagesController skips the policy and I'm explicitly testing that the pages controller permits caching since this only hosts static, non sensitive content.